tests/src/gui/testqgsscalecombobox.cpp

```diff
@@ -256,5 +257,33 @@ void TestQgsScaleComboBox::cleanup()
   delete s;
 }
 
+void TestQgsScaleComboBox::testLocale()
+{
+  QLocale::setDefault( QLocale::English );
+  QVERIFY( s->toString( 1e8 ) == "1:100,000,000" );
```

Comment:
Would be better to use QCOMPARE for these tests -- then you'd see exactly what the actual returned string is